src/core/js/models/adaptModel.js

```diff
@@ -606,26 +612,26 @@ define([
           console.warn("AdaptModel.checkLocking: unknown _lockType \"" +
```

Comment:
Suggest template literal:
```suggestion
          console.warn(`AdaptModel.checkLocking: unknown _lockType "${lockType}" found on ${this.get('_id')}`);
```

See also line 676.